financialaid/templatetags/financial_aid_tags.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Instead of defining these two template tags please do this calculation in the view and send the result to the template to be displayed.